### ISSUE TITLE
Can't run multiple commands again

### ISSUE THREAD

**Issue by Sir-Will** [lucko/LuckPerms#102]
hey,

running a command like /perm user ... set ... multiple times without a delay makes only one command work.
This issue was fixed before but seems to be back now.

**Comment by lucko** [comment]
Seems unlikely - I haven't changed any of that logic since fixing it.

**Comment by Sir-Will** [comment]
This shows up in the log after running 3 commands:
```
2016-12-23 22:22:34 | [LP] jackalots already has this permission!
2016-12-23 22:22:34 | [LP] Set some.permission to true for jackalots.
2016-12-23 22:22:34 | [LP] Permissions data could not be loaded. Please contact an administrator.
2016-12-23 22:22:34 | [LP] Permissions data could not be loaded. Please contact an administrator.
2016-12-23 22:22:34 | [LP] (User data was saved to storage)
```

**Comment by lucko** [comment]
Which platform / datastore.

**Comment by Sir-Will** [comment]
MySQL

**Comment by lucko** [comment]
Which platform?

**Comment by Sir-Will** [comment]
Spigot 1.10

**Comment by lucko** [comment]
Ok thanks - I'll look into it.

**Comment by lucko** [comment]
Oh, also, just to confirm, which version of LP were you running on that instance?

**Comment by Sir-Will** [comment]
2.14.20

**Comment by lucko** [comment]
You're running an old build which doesn't contain the fix for this issue.

Update to latest: https://ci.lucko.me/job/LuckPerms/

**Comment by Sir-Will** [comment]
Oh, sorry. Thought someone updated it on that server seems it got only updated on the servers. 

Am 24.12.2016 1:46 nachm. schrieb Luck <notifications@github.com>:

Closed #102.
